src/vma/sock/sockinfo_udp.cpp

```diff
@@ -469,6 +468,11 @@ int sockinfo_udp::bind(const struct sockaddr *__addr, socklen_t __addrlen)
 	on_sockname_change(name, *namelen);
 	si_udp_logdbg("bound to %s", m_bound.to_str());
 
+	if (m_rx_ring_map.size() == 1) {
+		rx_ring_map_t::iterator rx_ring_iter = m_rx_ring_map.begin();
+		m_p_rx_ring = rx_ring_iter->first;
```

Comment:
In the vma_get_socket_rings_fds()   get_rings_fds()  m_p_rx_ring is used and should be initialized  m_p_rx_ring->get_rx_channel_fds(). Please see sockinfo.h line 105.